### ISSUE TITLE
HACS Integration Improvements

### ISSUE THREAD

**Issue by Veldkornet** [MikeBrink/home-assistant-picnic#4]
Good work so far on the component, I understand it's still a work in progress, but I have noticed a few small items when it's installed via HACS (which are not really functional issues) but would be small improvements, if you were not already aware of these.

- From the "information" screen (ie, which you open the component in HACS) and you select "For more information visit the repository.", it redirects you [here](https://raw.githubusercontent.com/MikeBrink/home-assistant-picnic/) instead of to the repository
- When updating in HACS, there's no release notes to show what has changed since the previous version
- Version are git commit reverences instead of actual version numbers
- Ultimately, it would be nice if the integration were shown in the HA Integrations menu

**Comment by MikeBrink** [comment]
Hi,

Glad you are liking the integration!
Thanks for you multiple requests / issues. Il pick up this issue likely tomorrow and have a look at your requests the coming weekend.

**Comment by MikeBrink** [comment]
Can you elaborate on what you mean by:
"Ultimately, it would be nice if the integration were shown in the HA Integrations menu"

Do you mean [this](https://www.home-assistant.io/integrations/) page?

**Comment by Veldkornet** [comment]
No, I mean in HA. If you go to Configuration -> Integrations, most of the integrations are shown here (not all, most of the integrations are still being updated to be included here), with all of the related entities etc. It doesn't make things work any better, just that things are then more polished off ;)

An example of a HACS integration which does this is that of [UniFi Protect](https://github.com/briis/unifiprotect) or [Blitzortung](https://github.com/mrk-its/homeassistant-blitzortung)

**Comment by MikeBrink** [comment]
Hmm, weird. When I manually installed the integration it did show up. But when installing through HACS it does not show up as you say. I did not find a quick fix for this, so I'll have a look on this the comming weekend.
